inc/class-googleanalytics.php

```diff
@@ -339,52 +277,17 @@ public function displayBookAnalyticsSettings(): void {
 	}
 
 	public function printScripts(): void {
-		$network_google_v3_code = self::getGoogleIDSiteOption( 3, false );
-		if ( ! empty( $network_google_v3_code ) ) {
-			$book_google_v3_code = get_site_option( self::$is_allowed_option ) ?
-				$this->getGoogleIDSiteOption( 3, true ) : '';
+		$network_google_code = $this->getGoogleIDSiteOption( false );
+		if ( ! empty( $network_google_code ) ) {
+			$book_google_code = get_site_option( self::$is_allowed_option ) ?
+				$this->getGoogleIDSiteOption( true ) : '';
```

Comment:
Could we use named arguments here? I had to check the method to understand what `true|false` means in the context.

Reply:
it makes sense to me, it is not so intuitive. Thanks, Dalcin, I'll add it.